app/lib/service/security_advisories/backend.dart

```diff
@@ -79,6 +79,7 @@ class SecurityAdvisoryBackend {
       }
 
       if (!_isValidAdvisory(osv)) return null;
+      final syncTime = clock.now();
```

Comment:
I think this is sound, but it might be slightly more elegant to pass `syncTime` into `ingestSecurityAdvisory` such that all advisories from the same zip gets the same `syncTime`.

Reply:
done